config/routes.rb

```diff
@@ -1,11 +1,6 @@
 Rails.application.routes.draw do
   root to: 'dashboards#index', via: :get
   resource :dashboard, only: [:show]
-
-  get '/login', to: 'sessions#new'
-  post '/login', to: 'sessions#create'
-  get '/logout', to: 'sessions#destroy'
-
   resources :snippets, only: [:new, :create]
-  resources :users, only: [:new, :create]
+  resources :users, only: [:create]
```

Comment:
I get the following error when I remove line 5: `ActiveModel::ForbiddenAttributesError in Clearance::UsersController#create`